### PR TITLE
Make cmdline support third party modules

### PR DESCRIPTION
**Pull request by fyrestone**
## What do these changes do?

Support load third party modules for cmdline. The load modules include:
- The `--load-modules` args
- The `third_party_modules` section in config
- The `MARS_LOAD_MODULES` env

## Related issue number
N/A

### CODE REVIEW

File: mars/deploy/oscar/tests/test_cmdline.py [hunk 1]
```diff
@@ -179,7 +182,7 @@ def test_cmdline_run(supervisor_args, worker_args, use_web_addr):
         for proc in w_procs + [sv_proc]:
             if not proc:
                 continue
-            proc.terminate()
+            proc.send_signal(signal.SIGINT)
```
Comment (wjsi) [inline mars/deploy/oscar/tests/test_cmdline.py:185]: May not perform as expected in Windows.

Comment (fyrestone) [reply mars/deploy/oscar/tests/test_cmdline.py:185]: @keyile Why change this line?

Comment (fyrestone) [reply mars/deploy/oscar/tests/test_cmdline.py:185]: @wjsi @keyile I will revert this line to pass the Windows CI.

**Review by wjsi — APPROVED** [review]
LGTM

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**